generator.py

```diff
@@ -7,7 +7,7 @@
 import asyncio
 
 DEFAULT_GIR_FILES_DIRECTORY = Path("./gir-files")
-DEFAULT_GIR_DIRECTORY = Path("./gir/")
+DEFAULT_GIR_DIRECTORY = Path("./gir/") # TODO: This is typically the directory _of this Python script_ (which external projects symlink to)
```

Comment:
The TODO here means: should we come up with a different way, instead of "guessing/hardcoding" this _relative directory_?